shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/domain/usecase/LogInUseCase.kt

```diff
@@ -0,0 +1,12 @@
+package co.nimblehq.ic.kmm.suv.domain.usecase
+
+import co.nimblehq.ic.kmm.suv.domain.model.Token
+import co.nimblehq.ic.kmm.suv.domain.repository.UserRepository
+import kotlinx.coroutines.flow.Flow
+
+class LogInUseCase(private val repository: UserRepository) {
```

Comment:
Is it possible to add protocol for this usecase?

Reply:
@blyscuit I'm considering to add protocol if we can't mock this with iOS project. For android, we can mock with class directly, so [the android templates](https://github.com/nimblehq/android-templates/blob/develop/template/domain/src/main/java/co/nimblehq/template/domain/usecase/UseCase.kt) did not add protocol for usecase